cio/migrations/2023-08-29_add_scorers_to_read_applicants/up.sql

```diff
@@ -0,0 +1 @@
+CREATE VIEW read_applicants AS SELECT id, airtable_record_id, name, email, role, status, interviews_started, start_date, rejection_sent_date_time, offer_created, interested_in, question_why_oxide, submitted_time, location, linkedin, github, portfolio, website, resume, materials, portfolio_pdf, scorers FROM applicants;
```

Comment:
I think since this view already exists, you will need to have a `DROP ...` statement before the new `CREATE ...` 

```
DROP VIEW read_applicants;
CREATE VIEW read_applicants AS SELECT id, airtable_record_id, name, email, role, status, interviews_started, start_date, rejection_sent_date_time, offer_created, interested_in, question_why_oxide, submitted_time, location, linkedin, github, portfolio, website, resume, materials, portfolio_pdf, scorers FROM applicants;
```

Otherwise looks good.